srcpkgs/fluxbox/template

```diff
@@ -10,17 +11,21 @@ configure_args="--enable-nls
  ac_cv_lib_Xrandr_XRRQueryExtension=yes ac_cv_lib_fribidi_fribidi_version_info=yes"
 hostmakedepends="pkg-config"
 makedepends="libSM-devel libXrender-devel libXft-devel libXpm-devel imlib2-devel
- libXext-devel libXinerama-devel libXrandr-devel fribidi-devel"
+ libXext-devel libXinerama-devel libXrandr-devel fribidi-devel automake libtool
+ gettext gettext-devel"
```

Comment:
automake, libtool, and gettext (and maybe gettext-devel) should probably be hostmakedeps

Reply:
read but dont understand the diff. 1 for build and the other for some flag that installs sources for you or something?